demo/src/main/AndroidManifest.xml

```diff
@@ -8,8 +8,17 @@
         android:label="@string/app_name"
         android:theme="@style/AppTheme" >
         <activity
-            android:name=".PinDemoActivity"
-            android:label="@string/app_name" >
+                android:name=".PinDemoActivity"
+                android:label="@string/app_name" >
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
```

Comment:
Confusing to see two activities with the same intent filters. 
You should consolidate to one DemoActivity and have two buttons whether to inflate a support fragment or native fragment.